tests/profiles/cff_test.py

```diff
@@ -62,3 +62,20 @@ def test_check_cff2_call_depth():
     assert_results_contain(check(font),
                            FAIL, 'recursion-error',
                            'Recursion error while decompiling glyph "F".')
+
+
+def test_check_cff_deprecated_operators():
+    check = TestingContext(cff_profile,
+                           "com.adobe.fonts/check/cff_deprecated_operators")
```

Comment:
thanks a lot for adopting the new style of code tests ;-D

Reply:
Sure 😄